.changeset/twelve-coats-rush.md

```diff
@@ -0,0 +1,35 @@
+---
+'astro': major
+---
+
+The endpoints that have **lowercase** names are **deprecated**.
+
+The endpoints methods **need** to be in **uppercase**.
```

Comment:
I think we can rephrase this to emphasise the action users need to take, i.e. when thinking about a CHANGELOG as a guide that users need to act upon, we can focus on users _renaming_ functions rather than the passive state functions should have:

```suggestion
Lowercase names for endpoint functions are now deprecated.

Rename functions to their uppercase equivalent:
```